src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -117,21 +135,6 @@ impl LevelCompactionPicker {
         level_handlers: &[LevelHandler],
         stats: &mut LocalPickerStatistic,
     ) -> Option<CompactionInput> {
-        let l0_size = l0.total_file_size - level_handlers[0].get_pending_file_size();
```

Comment:
Will this  effect the strategy?

Reply:
This may affect performance, but there is no correctness issue and I wanted to prioritize simplifying the code. 

And will there be a new base level picker strategy?